.github/workflows/citizen.yml

```diff
@@ -0,0 +1,87 @@
+name: Citizen Sync
+
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Variable Settings
+      run: |
+        pr_num=$(echo ${GITHUB_REF} | cut -d "/" -f 3)
+        echo "::set-env name=LOG_PATH::https://github.com/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}"
+        echo "::set-env name=PR_PATH::https://github.com/${GITHUB_REPOSITORY}/pull/${pr_num}"
+
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+
+    - name: Install Third party libraries
+      run: |
+        python -V
+        sudo apt install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev automake libtool lsof figlet libsecp256k1-dev jq
+
+    - name: Run MQ
+      run: |
+        docker pull rabbitmq:3.7 
+        sudo docker run -d -p 5672:5672 -h localhost --name rabbit rabbitmq:3.7
+
+    - name: Install RewardCalculator
+      run: |
+        go version
+
+        git clone https://github.com/icon-project/rewardcalculator.git
+        cd rewardcalculator
+        git checkout develop -f
+        make && sudo make install
+        cd -
+        icon_rc -version
+
+    - name: Install Loopchain
+      run: |
+        pwd
+        make develop
+
+    - name: Generate-key
+      run: |
+        sed "23 a\  \"PRIVATE_PASSWORD\": \"${pwd}\"," -i conf/mainnet/loopchain_conf.json
+        cat conf/mainnet/loopchain_conf.json
+
+        pip install tbears
+        tbears keystore -p ${pwd} my_keystore.json
+      env:
+        pwd: Most_safest_pw_in_the_wor1d
```

Comment:
most safest! 😄

Reply:
> Is this test for mainnet sync while 20 seconds?

The test scenario is summarized:
- Run Loopchain to sync
- Give a short time to the node to **reach the target block height.**
- Ask to node whether it reached to the target or not.

The waiting time is essential to reach the target block height.
But there's no reason for the time to be `20` sec.

Reply:
Sorry. I mistaked opening this PR with wrong number.

Please see: https://github.com/icon-project/loopchain/pull/488